### ISSUE TITLE
original pretrained model modification

### ISSUE THREAD

**Issue by iliadsouti** [XiaohangZhan/mix-and-match#1]
For using the original pretrained models (jigsaw, colorize), they have to be modified (because of different BatchNorm/BN layer). Can you share your modification code? Or at least make some clues about how to do it?
